Plots/MapProjections/NCL_proj_2_lg.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 13 10:21:41 2020
+
+@author: misi1684
+"""
+
+
+import numpy as np
```

Comment:
Could you please consider separating significant code blocks (e.g. import packages, read data or generate data, plot, etc.) from each other by using special strings such as following:

```
###############################################################################
# Import packages:
```

FYI: Whenever you use the above exact amount of number sign (#) in a line, the following code block is separated as a notebook cell by Sphinx in RTD.